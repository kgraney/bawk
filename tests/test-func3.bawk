
a = 100;

def bar(c) {
	print(c);
}

def foo(a, b, c) {
	print (a);
	print (b);
	print (c);
	a = 10;
	bar(c * 2);
	print(a);
	print(c);
	c = c * 2;
	print(c);
}

foo(50+20, 10, a);
