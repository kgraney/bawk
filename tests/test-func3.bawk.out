70
10
100
200
10
100
200
